README.md

```diff
@@ -2,17 +2,18 @@
 
 The `bdd-for-c` library is a [BDD][bdd] test framework for the [C language][c].
 
+
 ## Quick Start
 
 To start, simply download the framework header file which can be easily done
-with curl on Linux and OS X:
+with curl on [MacOS][mac], [Linux][linux], and [BSD Unix][bsd]:
```

Comment:
Not sure it makes sense to link to distributions here. Maybe more helpful would be to tell people how to save it from the browser?

Reply:
I guess I'm not sure what you mean about "link to distributions".  Are you saying you don't think the links to the different OSes are useful, or are you saying something about the URI/link for curl?

I'm not really wedded to the OS links, so getting rid of those (or keeping them) is fine with me.  I might have been a little overly documentation-oriented when I made these edits, like back in the day when I wrote fourteen articles a month for TechRepublic.  Your call on keeping or discarding the links (obviously).

I think explaining how to save the file from the browser *in addition to* explaining how to get it with curl makes sense, though.

Reply:
Yes, sorry I meant that links to OSes are probably not useful as they won't really help identified what needs to be done and in general I would expect people to know which OS they are running.

And yes adding browser download instructions would be great, but I could also do it after merge the PR myself.